arangosh/Restore/RestoreFeature.cpp

```diff
@@ -588,8 +625,16 @@ arangodb::Result processInputDirectory(
       }
     }
     std::sort(collections.begin(), collections.end(), ::sortCollections);
-
-    // Step 2: run the actual import
+    
+    // Step 2: recreate all views
+    for (VPackBuilder viewDefinition : views) {
```

Comment:
it might be necessary to recreate views before collections or collection indexes might fail during create

Reply:
Collection are recreated in step 3.